src/vs/workbench/contrib/notebook/browser/contrib/chatEdit/notebookSynchronizer.ts

```diff
@@ -249,7 +249,7 @@ export class NotebookModelSynchronizer extends Disposable {
 	private _originalModel?: Promise<NotebookTextModel>;
 	private async getOriginalModel(entry: IModifiedFileEntry): Promise<NotebookTextModel> {
 		if (!this._originalModel) {
-			this._originalModel = this.originalModelRefFactory.getOrCreate(entry, this.model.viewType).then(ref => this._register(ref).object);
+			this._originalModel = this.originalModelRefFactory.getOrCreate(entry as unknown as any, this.model.viewType).then(ref => this._register(ref).object);
```

Comment:
This file will be deleted in a separate PR, hence the dirty cast to avoid compilation errors.
Wanted to minize diffs in this PR hence not removed here.